lib/suspenders/actions.rb

```diff
@@ -15,7 +15,7 @@ def action_mailer_host(rails_env, host)
     end
 
     def action_mailer_asset_host(rails_env, host)
-      config = "config.action_mailer.asset_host = { host: #{host} }"
+      config = "config.action_mailer.asset_host = #{host}"
```

Comment:
It could have quotes, but it has not for the production environment.
The `%{}` has been used in https://github.com/thoughtbot/suspenders/blob/master/lib/suspenders/app_builder.rb#L301 so I think we are good here